data/settings/1.15.x/dns.toml

```diff
@@ -0,0 +1,19 @@
+[[docs.ref.name-servers]]
+description = "A list of IP address strings that represent the desired name server(s)."
+accepted_values = [ "IP addresses" ]
+see = [
+    ["[`resolve.conf` manual page](https://man7.org/linux/man-pages/man5/resolv.conf.5.html)"]
+]
+note = "You must provide both this setting and [`settings.dns.search-list`](#search-list) otherwise Bottlerocket gathers the DNS search list and name servers from the DHCP lease."
+[[docs.ref.name-servers.example]]
+value = "[\"1.2.3.4\", \"5.6.7.8\"]"
```

Comment:
I don't think we want to encourage these settings to be specified together. They can be, but they serve different purposes and can be used independently:
* If the DNS search list setting is not set, the DHCP value is used
* If the name servers setting is not set, the DHCP value is used

Reply:
Oh interesting!

From the readme docs:
"If **either** DNS setting is not populated, the system will use the DHCP lease of the primary interface to gather **these settings**."

I took this to mean "if you don't have both, we use the stuff from DHCP." I guess that's wrong!